beam-core/Database/Beam/Backend/SQL/SQL2003.hs

```diff
@@ -34,6 +35,10 @@ class IsSql92FromSyntax from =>
                     -> Maybe Text
                     -> from
 
+class IsSql92OrderingSyntax ord =>
+    IsSql2003OrderingSyntax ord where
+    nullsFirst, nullsLast :: ord -> ord
```

Comment:
To be in line with the naming convention in `IsSql92OrderingSyntax` can we rename these `nullsFirstOrdering` and `nullsLastOrdering`?